modmail/bot/bot.py

```diff
@@ -62,6 +62,10 @@ async def on_ready(self) -> None:
 
     async def on_message(self, message: discord.Message) -> None:
         if message.guild and message.guild.me in message.mentions:
+            member = self.guild.get_member(message.author.id)
+            if Config.STAFF_ROLE_ID in [role.id for role in member.roles]:
```

Comment:
You should be loading the role in the `on_ready` using this config ID, then checking if the role object is currently in the member roles.